library.properties

```diff
@@ -1,5 +1,5 @@
 name=Exosite
-version=2.5.3
+version=2.6.1
 author=Applications Engineering <support@exosite.com>
 maintainer=Thane Storley <thanestorley@exosite.com>
```

Comment:
Is Thane the maintainer?  Or should we set this to Support as well?

Reply:
I updated it to be the same as author.  Accidentally did it on master so it's already in.